### PR TITLE
frontend: try to load apropriate metadata files for ardupilot

### PR DESCRIPTION
**Pull request by Williangalvani**
these should meaningfully update the metadata for versions higher than 4.1

### CODE REVIEW

File: core/frontend/src/types/autopilot/parameter-table.ts [hunk 1]
```diff
@@ -94,29 +97,45 @@ export default class ParametersTable {
...
   async fetchMetadata(): Promise<void> {
     if (autopilot.vehicle_type === null) {
       // Check again later if we have a vehicle type identified
       fetchVehicleType()
       setTimeout(() => { this.fetchMetadata() }, 1000)
       return
     }
-
-    if (autopilot_data.autopilot_type === MavAutopilot.MAV_AUTOPILOT_PX4) {
-      this.metadata = fromPX4toArduPilotParametersMetadata(await fetchPX4Metadata())
```
Comment (JoaoMario109) [inline core/frontend/src/types/autopilot/parameter-table.ts:106]: I think that the PX4 case was removed by mistake

File: core/frontend/vite.config.js [hunk 2]
```diff
@@ -48,7 +48,7 @@ export default defineConfig({
       version: 2.7,
     }),
   ],
-  assetsInclude: ['**/*.gif', '**/*.glb', '**/*.png', '**/*.svg'],
+  assetsInclude: ['**/*.gif', '**/*.glb', '**/*.png', '**/*.svg', '**/assets/ArduPilot-Parameter-Repository**.json'],
```
Comment (patrickelectric) [inline core/frontend/vite.config.js:51]: Did you test this through a docker release ?

Comment (Williangalvani) [reply core/frontend/vite.config.js:51]: yep

File: core/frontend/src/types/autopilot/parameter-table.ts (hunk 1 above)
Comment (Williangalvani) [reply core/frontend/src/types/autopilot/parameter-table.ts:106]: oh no, yeah, definitely a mistake :p

**Review by patrickelectric — APPROVED** [review]

**Pull request merged**